flang/include/flang/Runtime/descriptor.h

```diff
@@ -420,6 +420,27 @@ class Descriptor {
 
   void Dump(FILE * = stdout) const;
 
+// Number of bits used to encode the addendum presence flag.
+#define _CFI_ADDENDUM_BITS 1
+// Value of the addendum presence flag.
+#define _CFI_ADDENDUM_FLAG 1
+// Allocator index mask.
+#define _CFI_ALLOCATOR_IDX_MASK 0b00001110
+
+  RT_API_ATTRS inline bool HasAddendum() const {
+    return raw_.extra & _CFI_ADDENDUM_FLAG;
+  }
+  RT_API_ATTRS inline void SetHasAddendum() {
+    raw_.extra |= _CFI_ADDENDUM_FLAG;
+  }
+  RT_API_ATTRS inline int GetAllocIdx() const {
+    return (raw_.extra & _CFI_ALLOCATOR_IDX_MASK) >> _CFI_ADDENDUM_BITS;
```

Comment:
One last tweak, then I think I'm done: please rename _CFI_ADDENDUM_BITS, at least as it is used here, to something that signifies that it's the shift count for the ALLOCATOR_IDX field.  That will make these two member functions independent of any assumption that the ALLOCATOR_IDX field has to be adjacent to the addendum flag and that both are right-justified.

Reply:
Done. Thanks for the review!